Sources/PublishCLICore/WebsiteRunner.swift

```diff
@@ -55,6 +65,13 @@ private extension WebsiteRunner {
         catch { throw CLIError.outputFolderNotFound }
     }
 
+    /// The major version number of the current system's Python install (whatever Python responds to the `python` command)
+    var systemPythonMajorVersionNumber: Int? {
```

Comment:
Like mentioned above, I'd make this a method instead, since it actually performs XPC. Giving it property semantics sends the wrong message, IMO, since you'd assume that you can access it any number of times without a performance penalty, which isn't the case.

Reply:
I like this. I've never really had a litmus test for when to use a variable versus a function for a simple method without arguments, but thinking of a performance penalty as the criteria is good. I'll make the change.